core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2363,4 +2363,12 @@ package object config {
       .version("3.3.0")
       .intConf
       .createWithDefault(5)
+
+  val SPARK_PROPERTIES_KEY_QUEUE = ConfigBuilder("spark.properties.key.queue")
```

Comment:
The configuration namespace looks a little misleading to me if this is only applicable to YARN.

Reply:
How does `spark.queue.key` or `spark.key.queue` sound?

Reply:
> I'm not sure about adding an empty column to UI to all resource managers. This had better be disabled by default.
> 
> cc @gengliangwang too because this is a breaking UI change.

Even though I believe Queue information is very common in the platforms running Spark, we could start with only showing the queue column if the configuration `spark.properties.key.queue` is specified.